spec/spec_helper.rb

```diff
@@ -83,3 +83,88 @@ def eql?(other)
     true
   end
 end
+
```

Comment:
Rather than just committing the default `spec_helper` generated by `rspec --init`, I wonder if it would be better to trim this down. At least remove the verbose comments.